lib/logflare/endpoints.ex

```diff
@@ -159,10 +160,21 @@ defmodule Logflare.Endpoints do
     %Query{query: query_string, user_id: user_id, sandboxable: sandboxable} = endpoint_query
     sql_param = Map.get(params, "sql")
 
-    transform_input =
-      if(sandboxable && sql_param, do: {query_string, sql_param}, else: query_string)
+    endpoints =
+      list_endpoints_by(user_id: endpoint_query.user_id)
+      |> Enum.filter(&(&1.id != endpoint_query.id))
+
+    alerts = Alerting.list_alert_queries_by_user_id(endpoint_query.user_id)
 
     with {:ok, declared_params} <- Logflare.Sql.parameters(query_string),
+         {:ok, expanded_query} <-
+           Logflare.Sql.expand_subqueries(
+             endpoint_query.language,
+             query_string,
+             endpoints ++ alerts
+           ),
+         transform_input <-
+           if(sandboxable && sql_param, do: {expanded_query, sql_param}, else: expanded_query),
```

Comment:
```suggestion
         transform_input =
           if(sandboxable && sql_param, do: {expanded_query, sql_param}, else: expanded_query),
```

Or even move it into `do` block, as it is not needed in the `with` patterns.

Reply:
we want to catch any errors in the subsequent `Sql.transform/3` call, hence why it can't be moved into `do`